packages/hadron-document/src/editor/date.js

```diff
@@ -1,13 +1,7 @@
-import moment from 'moment-timezone';
```

Comment:
Same as with `BSONValue` we never pass a custom timezone to the date, it's always UTC so I'm removing all the code that handles timezone until we want to actually support this